common/src/uint/mod.rs

```diff
@@ -41,3 +40,101 @@ pub trait BitArray {
     /// Create value representing one
     fn one() -> Self;
 }
+
+#[derive(Debug, Clone)]
+pub struct U256Encodable(pub U256);
```

Comment:
Instead of this, maybe we should just use H256 and convert back and forth.

And btw, it's always preferable to use From+Into+Deref+AsRef instead of making the inner a `pub`.